util/topgen/strong_random.py

```diff
@@ -74,7 +76,7 @@ def fetchbyte(self):
             sys.exit(1)
         else:
             random_byte = self.buffer[0]
```

Comment:
Now that it's a normal Python array, you can probably replace this line and the next with:
```suggestion
            random_byte = self.buffer.pop(0)
```

Reply:
This is indeed better. Thanks. It's fixed now.